spec/langchain/llm/ollama_spec.rb

```diff
@@ -31,7 +31,7 @@
       response = subject.complete(prompt: "In one word, life is ")
 
       expect(response).to be_a(Langchain::LLM::OllamaResponse)
-      expect(response.completion).to eq("\nIn one word, life is... complex.")
+      expect(response.completion).to eq("fragile.")
```

Comment:
@drnic I guess an LLM could never relate to the fact that, also, "life is beautiful" 😄